Remotion/Web/Development.WebTesting.IntegrationTests/Web.Development.WebTesting.IntegrationTests.csproj

```diff
@@ -21,5 +21,8 @@
     <ProjectReference Include="..\Development.WebTesting.WebFormsControlObjects\Web.Development.WebTesting.WebFormsControlObjects.csproj" />
     <ProjectReference Include="..\Development.WebTesting\Web.Development.WebTesting.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Selenium.WebDriver" Version="4.5.0" />
+  </ItemGroup>
```

Comment:
Why do we need to add this reference? it Wasn't needed before the version upgrade.